qiskit/providers/aer/backends/__init__.py

```diff
@@ -19,3 +19,5 @@
 from .statevector_simulator import StatevectorSimulator
 from .unitary_simulator import UnitarySimulator
 from .pulse_simulator import PulseSimulator
+
+from .aer_compiler import AerCompiler
```

Comment:
This should just be an internal part of the AerBackend so it happens automatically during the `run` call, but I think this requires qiskit terra transpiler support for the qasm3 instructions first.

Reply:
I removed `AerCompiler` from `__init__.py`.

Reply:
I removed `AerCompiler`.